cmd/cli/mesh_list.go

```diff
@@ -77,5 +71,13 @@ func (l *meshListCmd) run() error {
 	fmt.Fprint(w, getPrettyPrintedMeshInfoList(meshInfoList))
 	_ = w.Flush()
 
+	fmt.Fprintf(l.out, "\nTo list the OSM controller pods, please run the following command\n")
+	fmt.Fprintf(l.out, "\tkubectl get pods -n %s -l app=osm-controller\n", settings.Namespace())
```

Comment:
Should the namespace for this command be generic ( <mesh namespace> ) since `settings.Namespace()` might not be the namespace of the mesh the user is trying to get the controller pods from? I think what you have above makes sense too

Reply:
@jaellio Resolved - Instructed the user to instead put the namespace of the mesh they want to list.

```
	fmt.Fprintf(l.out, "\nTo list the OSM controller pods for a mesh, please run the following command passing in the mesh's namespace\n")
	fmt.Fprintf(l.out, "\tkubectl get pods -n <osm-mesh-namespace> -l app=osm-controller\n")
```